experimental/generation/generator/packages/library/templates/standard/library-SkipIntent.dialog.lg

```diff
@@ -0,0 +1,31 @@
+[generator.lg](../generator.lg)
+
+# template
+- ```
+{
+    "$schema": "${appSchema}",
+    "$kind": "Microsoft.OnIntent",
+    "intent": "Skip",
```

Comment:
When skip intent is needed ?

Reply:
This allows you to skip the current ambiguity event or property being changed.

---
In reply to: [478255038](https://github.com/microsoft/BotBuilder-Samples/pull/2690#discussion_r478255038) [](ancestors = 478255038)